pulsar-client/src/main/resources/ahc.properties

```diff
@@ -0,0 +1,44 @@
+com.yahoo.pulsar.shade.org.asynchttpclient.threadPoolName=AsyncHttpClient
```

Comment:
Can you add a comment here explaining what's the purpose of this file? Same thing that you explain in the description